assets/src/dashboard/app/views/myStories/content/storiesView/index.js

```diff
@@ -267,10 +267,29 @@ function StoriesView({
                   setFocusedStory({ id: activeStory.id });
                   setActiveDialog('');
                 }}
+                aria-label={sprintf(
+                  /* translators: %s is story title */
+                  __(
+                    'No, I do not want to delete the story titled: %s',
```

Comment:
Using full sentences seems a bit out of place. Especially odd when read aloud by a screen reader.

Something like "Cancel deleting story "xyz" and "Confirm deleting story xyz" does not work?

Reply:
sounds good, i'll update - thanks for the suggestion!